handshake_server.go

```diff
@@ -69,6 +69,7 @@ func (c *Conn) serverHandshake() error {
 	// For an overview of TLS handshaking, see https://tools.ietf.org/html/rfc5246#section-7.3
```

Comment:
Also this comment is dumb and outdated. obviously handshaking is described in some section of TLS 1.2 and 1.3 RFC